emcc.py

```diff
@@ -182,7 +182,16 @@ def run():
     exit(0)
 
   elif '--cflags' in sys.argv:
-    print ' '.join(shared.COMPILER_OPTS + ['-emit-llvm', '-c'])
+    # fake running the command, to see the full args we pass to clang
+    debug_env = os.environ.copy()
+    debug_env['EMCC_DEBUG'] = '1'
+    args = filter(lambda x: x != '--cflags', sys.argv)
+    out, err = subprocess.Popen([shared.PYTHON] + args + [shared.path_from_root('tests', 'hello_world.c'), '-c'], stderr=subprocess.PIPE, env=debug_env).communicate()
```

Comment:
Yeah, you're right, thanks. Sorry, I'm quite sick atm and probably shouldn't be writing code... last commit should fix that.